google-cloud-storage/src/main/java/com/google/cloud/storage/RetryableInputStreamContent.java

```diff
@@ -0,0 +1,49 @@
+package com.google.cloud.storage;
+
+import com.google.api.client.http.AbstractInputStreamContent;
+import sun.nio.ch.IOUtil;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public final class RetryableInputStreamContent extends AbstractInputStreamContent {
```

Comment:
Please check the comment for `HttpStorageRpc.create()`. There is a chance that you do not need this class.